api-tests/dev_apis/storage/test_s008/test_data.h

```diff
@@ -40,7 +40,7 @@ static const test_data_t s008_data[] = {
 },
 {
     /* Call get API with data_len = MAX_UINT32 */
-    VAL_TEST_IDX5, {VAL_ITS_GET, VAL_PS_GET}, PSA_ERROR_INVALID_ARGUMENT
+    VAL_TEST_IDX5, {VAL_ITS_GET, VAL_PS_GET}, PSA_ERROR_PROGRAMMER_ERROR
```

Comment:
You should not assume this is the error code. It is the result that TF-M will provide as the buffer is validated by the FF-M framework implementation. In general, this could be other errors (or even a Fault).

I would suggest checking that the result is not `PSA_SUCCESS` rather than checking for a specific error.

Reply:
Updated the commit as requested.
Please provide further comments, if any.